prow/plugins/branch-updater/branch-updater.go

```diff
@@ -82,55 +82,67 @@ func handlePR(gc githubClient, log *logrus.Entry, config plugins.BranchUpdater,
 	repo := event.Repo.Name
 	pr := event.PullRequest
 
+	logger := log.WithFields(logrus.Fields{
+		"org":            org,
+		"repo":           repo,
+		"pr":             pr,
+		"mergeable":      *pr.Mergable,
+		"mergeableState": pr.MergeableState,
+	})
+
 	// Find the Tide status.
 	tideStatus, err := findTideStatus(gc, org, repo, pr)
 	if err != nil {
-		log.WithError(err).Errorf("Failed to get the context statuses on %s/%s#%d.", org, repo, pr.Number)
+		logger.WithError(err).Errorf("Failed to get context statuses for PR.")
```

Comment:
Are we not losing on the detail here by removing the template?
Same goes for the added logger lines. The PR number that the branch-updater is skipping is useful?

Reply:
Ahhhh perfect. I knew that I was missing something obvious.

Reply:
(I did just fix the `pr` field there to _just_ hold the PR number though)